packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -7784,7 +7784,10 @@ class IndexedSemantics extends SingleChildRenderObjectWidget {
 ///
 /// Useful for attaching a key to an existing widget.
 class KeyedSubtree extends StatelessWidget {
-  /// Creates a widget that builds its child.
+  /// Creates a widget that builds its child while assigning it a key.
+  ///
+  /// This is useful when you want to preserve the state of a widget when it moves
+  /// around in the widget tree by associating it with a consistent key.
```

Comment:
Just double checking, but did you mean to include this docs change in this PR? Fine either way as it looks like an improvement as well.

Reply:
I looks like these are from https://github.com/flutter/flutter/pull/167198

Reply:
https://github.com/flutter/flutter/pull/167198 Is approved to land, then we can return here. This still needs to be cleaned up.